### PR TITLE
No TOR browser allowed

### PR DESCRIPTION
**Pull request by valentinoConti**
### Description

We should not allow TOR Browser users to use Portal Bridge for regulatory and security reasons. This PR does that.

### Screenshots

Google Chrome
chrome

Tor Browser
tor

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -174,9 +175,59 @@ const useStyles = makeStyles((theme) => ({
...
 }));
 
 function App() {
+  const [isBlockedIP, setIsBlockedIP] = useState(false);
+
+  const checkForTOR = async () => {
+    // if not on mainnet, no need to check for TOR
+    if (process.env.REACT_APP_CLUSTER !== "mainnet") {
+      return false;
+    } else {
```
Comment (raop155) [inline src/App.js:203]: Maybe we can do this way:

```suggestion
    if (process.env.REACT_APP_CLUSTER === "mainnet") {
      ...check tor ips
    } 
```

**Review by raop155 — APPROVED** [review]
LGTM 🚀

File: src/App.js (hunk 1 above)
Comment (valentinoConti) [reply src/App.js]: The else was not needed, the second if neither 🙌🏼

**Comment by valentinoConti** [comment]
-> This will be fixed by the Infra team using CloudFlare Geo-IP blocking ("T1" is a country code CF have for Tor IPs).

**Pull request closed without merging**